floss/main.py

```diff
@@ -556,9 +565,12 @@ def main(argv=None) -> int:
         or results.analysis.enable_stack_strings
         or results.analysis.enable_tight_strings
     ):
-        if os.path.getsize(sample) > MAX_FILE_SIZE:
-            logger.error("cannot deobfuscate strings from files larger than 0x%x bytes", MAX_FILE_SIZE)
-            return -1
+        if sample_size > MAX_FILE_SIZE:
+            if not args.large_file:
+                logger.error("cannot deobfuscate strings from files larger than 0x%x bytes", MAX_FILE_SIZE)
+                return -1
+            else:
+                logger.warning("a large file was provided with a size of %i bytes, this may take much more time to process and require more system resource", sample_size)
```

Comment:
i like this message that indicates "even though you provided the flag, remember this might take a while"

Reply:
Opened PR #568 ! 😊